content/applications/finance/accounting/fiscal_localizations/localizations/germany.rst

```diff
@@ -31,7 +31,134 @@ Export from Odoo to Datev
 It is possible to export your accounting entries from Odoo to Datev. To be able to use this
 feature, the german accounting localization needs to be installed on your Odoo Enterprise database.
 Then you can go in :menuselection:`Accounting --> Reporting --> General Ledger` then click on the
-**Export Datev (csv)** button. 
+**Export Datev (csv)** button.
+
+.. _germany/pos:
+
+Point of Sale in Germany: Technical Security System
+===================================================
+
+Since the new regulations "The Act on Protection against Manipulation of Digital Records"
+(Kassensicherungsverordnung), a **Technical Security System** (also called **TSS** or **TSE**) needs
+to be used for each :doc:`point of sale </applications/sales/point_of_sale>`.
```

Comment:
```suggestion
With the introduction of new regulations, namely "The Act on Protection against Manipulation of Digital Records"
(Kassensicherungsverordnung), every :doc:`point of sale </applications/sales/point_of_sale>` must be equipped
**Technical Security System** (also called **TSS** or **TSE**).
```
Rephrased for clarity (just a suggestion - feel free to discard)

Reply:
Here is another phrasing that doesn't include a "new" aspect. It's just a fact, independent of the time of writing.

```suggestion
The **Kassensicherungsverordnung** (The Act on Protection against Manipulation of Digital Records)
requires that electronic record-keeping systems - including the :doc:`point of sale
</applications/sales/point_of_sale>` systems - must be equipped with a **Technical Security System**
(also called **TSS** or **TSE**).
```

Reply:
@xpl-odoo